lnbits/core/views/api.py

```diff
@@ -504,9 +508,21 @@ async def api_list_currencies_available():
 
 
 class ConversionData(BaseModel):
-    unit: str
+    unit: str = Query('sat')
     amount: float
+    to: str = Query('usd')
```

Comment:
can we have `from` and `to` just to be sane? for backwards compatibility is it possible to default `from` to `unit`?

Reply:
fixed, i think this is not being used so no backward compatibilty needed! 
`from` is a reserved keyword, I've set an alias on pydantic model.